index.html

```diff
@@ -4,13 +4,22 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/queries.css" />
     <title>Portfolio Haliljon</title>
   </head>
   <body>
+    <div class="all">
     <header class="header">
       <div class="container">
         <a href="#" class="link"> <strong>Welcome</strong></a>
+        <div class="nav-bar">
+          <ul>
+            <li class="nav-list">Portfolio</li>
+            <li class="nav-list">About</li>
+            <li class="nav-list">Contact</li>
```

Comment:
- [ ] [OPTIONAL] Great job adding the menu bar 🥇, but I think it could be better if you inside of each `<li>` tag insert a link `<a>` tag, these links should redirect to another section in your project. 🐰